common/src/models/domain/globalState.ts

```diff
@@ -21,4 +21,7 @@ export class GlobalState {
   noAutoPromptBiometrics?: boolean;
   noAutoPromptBiometricsText?: string;
   stateVersion: number;
+  environmentUrls?: any = {
+    server: "bitwarden.com",
+  };
```

Comment:
Does this mean we can't have different environments per account?

Reply:
Temporarily. Right now setting environment urls is just plain not working, moving them out of account scope is a temporary fix until I'm back next week to resolve the underlying issue with having them be scoped to an account, which is going to require a bit of thought.

Reply:
Fair enough, just wanted to verify the purpose.